src/test/java/javaslang/control/OptionTest.java

```diff
@@ -131,15 +133,15 @@ public void shouldBePresentOnIsEmptyWhenValueIsPresent() {
 
     @Test
     public void shouldConsumePresentValueOnIsPresentWhenValueIsPresent() {
-        final int[] actual = new int[] { -1 };
+        final int[] actual = new int[]{-1};
```

Comment:
The first thing I see is that you probably use IntelliJ with the default settings because the format changed slightly. Don't mind - it is ok! I provided the Javaslang settings in `./ide/idea-settings.jar`. If you use them it is good, if not it does not really matter...